### DISCUSSION TITLE
Enhancement: Improve the scoring system

### DISCUSSION THREAD

**Discussion by VSchris-code** [FunkinCrew/Funkin#1672] [Ideas]
Well, since the current scoring system is just plain bad, since it's just 350s, 200s, 100s, 50s, et cetera for hitting notes
Why not add different base scores per song and a increase in points earned for higher combo like in Taiko no Tatsujin?
I'd also let you comment of the ideas to improve the scoring system, just i case what I suggested doesn't fit well enough.
Also reccomend a grading system that goes like this, best to worst:
`SSS, SS, S, AAA, AA, A, B, C, D, E (if blue balled) - Suffix current grade except SSS with colon (:) if close to getting better grade, prefix with colon (:) if close to getting worse grade (forgot to add this in)`

**Comment by maska228** [comment]
basically mic'd up scoring system but complexier

**Comment by inue-kang** [comment]
Ye
